plugins/discovery-azure/src/test/java/org/elasticsearch/cloud/azure/AbstractAzureComputeServiceTestCase.java

```diff
@@ -48,7 +48,7 @@ protected Settings nodeSettings(int nodeOrdinal) {
 
         // We add a fake subscription_id to start mock compute service
         builder.put(Management.SUBSCRIPTION_ID, "fake")
-            .put(Discovery.REFRESH, "5s")
+            .put(Discovery.REFRESH_SETTING.getKey(), "5s")
```

Comment:
I think I was not really woke up. I thought it was a test assert...

```java
assertThat(Discovery.REFRESH_SETTING.getKey(), is("5s"));
```

Stupid me!  :)